tethys_apps/context_processors.py

```diff
@@ -41,4 +41,20 @@ def tethys_apps_context(request):
             if hasattr(app, 'enable_feedback'):
                 context['tethys_app']['enable_feedback'] = app.enable_feedback
 
+    # Dependency Versions
```

Comment:
I don't understand context processors as well as I should, but this is basically a pre-processor for all contexts before a template gets rendered, right? That's a good place to put this stuff. 👍

Reply:
Yes, that's the basic idea. I toyed with the idea of having the version in the settings.py... but I'm not sure it will always be as easy as switching the versions so I put it in there for now.